internal/proto/index_coord.proto

```diff
@@ -231,15 +231,16 @@ message StorageConfig {
     string access_keyID = 2;
     string secret_access_key = 3;
     bool useSSL = 4;
-    string bucket_name = 5;
-    string root_path = 6;
-    bool useIAM = 7;
-    string IAMEndpoint = 8;
-    string storage_type = 9;
-    bool use_virtual_host = 10;
-    string region = 11;
-    string cloud_provider = 12;
-    int64 request_timeout_ms = 13;
+    string sslCACert = 5;
```

Comment:
you can not add sslCACert in the middle of proto file since it will break compatibility .

add new params in the end is safe